site/docs/v1/tech/email-templates/templates-replacement-variables.adoc

```diff
@@ -0,0 +1,610 @@
+---
+layout: doc
+title: Email Templates and Variables
```

Comment:
Do we want this to reflect the same title we have in doc.liqued/navigation bar? 
"Email Templates and Variables" => "Email Variables"

I realize both templates and variables are related so I could be convinced otherwise. I do think the sub-heading "Templates and Replacement variables" suffices though.